sentry_sdk/integrations/aws_lambda.py

```diff
@@ -234,16 +233,29 @@ def _make_request_event_processor(aws_event, aws_context, configured_timeout):
 
     def event_processor(event, hint, start_time=start_time):
         # type: (Event, Hint, datetime) -> Optional[Event]
-        extra = event.setdefault("extra", {})
+        total_memory, used_memory, free_memory = map(
+            int, os.popen("free -t -m").readlines()[-1].split()[1:]
+        )
         remaining_time_in_milis = aws_context.get_remaining_time_in_millis()
         exec_duration = configured_timeout - remaining_time_in_milis
+
+        contexts = event.setdefault("contexts", {})
+        if (
+            isinstance(contexts, dict)
+            and "memory usage and execution duration" not in contexts
+        ):
+            contexts["memory usage and execution time"] = {
```

Comment:
I didn't find docs suggesting against spaces in the context key, but in practice we tend not to use it.

This information seems like a very good fit to the standard `runtime` context.
https://develop.sentry.dev/sdk/event-payloads/contexts/#runtime-context

Reply:
@rhcarvalho 
As per a suggestion given by AJ, I figured that a separate section for execution time and memory usage should be a good implementation. Also, the "Additional data" section which contains all data related to AWS Lambda contains space in the context key, and considering the visual aspect for UI, I kept spaces. 
So, as a solution, should I keep the execution time and memory sage inside lambda details or keep the current implementation (I'll share the screenshot of current implementation on slack)?

Reply:
>Also, the "Additional data" section which contains all data related to AWS Lambda contains space in the context key, and considering the visual aspect for UI, I kept spaces.

Let's stay consistent within the integration for now, but just as a side note rodolfo is right that most other integrations don't do this. It also leads to poor (or at least worse) UX when attempting to configure some other features such as server-side data scrubbing.

Reply:
So, I'll put the Execution time & memory usage data in the "Additional Data" section inside the "lambda" context.

Reply:
yes, just put it into `extra['lambda']`, you're right. Sorry about the confusion